library/src/main/java/com/firebase/ui/com/firebasei/ui/authimpl/TwitterAuthHelper.java

```diff
@@ -0,0 +1,61 @@
+package com.firebase.ui.com.firebasei.ui.authimpl;
+
+/**
```

Comment:
Please remove this ownership comment. You might want to change the template in Android Studio to not include to begin with.